### PR TITLE
sci-biology/fastp: add ebuild

### PR DESCRIPTION
**Pull request by mschubert**
A widely used nucleotide sequence trimmer

### CODE REVIEW

File: sci-biology/fastp/fastp-0.23.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+EAPI=8
+
+DESCRIPTION="An ultra-fast all-in-one FASTQ preprocessor"
+HOMEPAGE="https://github.com/OpenGene/fastp"
+SRC_URI="https://github.com/OpenGene/fastp/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
```
Comment (AndrewAmmerlaan) [inline sci-biology/fastp/fastp-0.23.4.ebuild:13]: Please drop empty vars

File: sci-biology/fastp/fastp-0.23.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+DEPEND="app-arch/libdeflate
+	dev-libs/isa-l"
+
+src_install() {
+	dodir /usr/bin
```
Comment (AndrewAmmerlaan) [inline sci-biology/fastp/fastp-0.23.4.ebuild:19]: `dodir` should not be necessary here.

Comment (mschubert) [reply sci-biology/fastp/fastp-0.23.4.ebuild:19]: Without this `make -j8 install` fails:
> install: cannot create regular file '/dev/shm/portage/sci-biology/fastp-0.23.4/image/home/m.schubert/gentoo/usr/bin/fastp': No such file or directory

**Comment by mschubert** [comment]
Removed empty var and added signoff

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**